brahe/frames.py

```diff
@@ -89,7 +90,7 @@ def polar_motion(epc:Epoch) -> np.ndarray:
     # Compute transformation and return
     rpm = _sofa.Pom00(xp, yp, _sofa.Sp00(_constants.MJD_ZERO, epc.mjd(tsys="TT")))
```

Comment:
Comment out the SOFA calls in this and above so we don't do the computation, if we're doing the hack let's get the benefit of making it run faster.